src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -563,7 +563,7 @@ protected function checkExistingCustomerBasket($customerId)
     {
         $basket = $this->basketManager->findOneBy(['customer' => $customerId]);
         if ($basket instanceof BasketInterface) {
-            throw new HttpException('400', sprintf('Customer (%d) already has a basket', $customerId));
+            throw new HttpException(400, sprintf('Customer (%d) already has a basket', $customerId));
```

Comment:
Please move this to a separate commit, it does not have to do with the core bundle, same for any unrelated change below.

Reply:
I did it for you, and actually made 4 commits.